README.md

```diff
@@ -56,20 +56,19 @@ If you want to see traces of the interactiosn for debugging and performance anal
 install the `jaeger-all-in-one` binary and the `jupyterlab-server-proxy-saulshanabrook`
 lab extension to see the Jaeger icon in the launcher.
 
-
 ```bash
-conda install -c conda-forge
+conda install jaeger -c conda-forge
```

Comment:
I think on the latest PR the `jaeger` was missing @saulshanabrook

Reply:
Ah thanks!